app/dashboard/static/js/app/view-tests-plan-id.2020.10.js

```diff
@@ -128,7 +128,7 @@ require([
         defconfigLink = document.createElement('a');
         defconfigLink.appendChild(
             document.createTextNode(results.defconfig_full));
-        defconfigPath = translatedURI[1] + "/kernel.config";
+        defconfigPath = translatedURI[1] + "/config/kernel.config";
```

Comment:
Tested OK on staging:
https://staging.kernelci.org/test/plan/id/60b85c1c157aa0107d902fc8/
which shows a link to:
https://storage.staging.kernelci.org/mainline/master/v5.13-rc4-61-g324c92e5e0ee/arm64/defconfig/clang-10/config/kernel.config